app/src/main/res/xml/customization_preferences.xml

```diff
@@ -12,23 +12,27 @@
         <org.mozilla.fenix.settings.RadioButtonPreference
             android:defaultValue="@bool/underAPI28"
             android:key="@string/pref_key_light_theme"
+            android:layout="@layout/preference_widget_radiobutton"
```

Comment:
Is this needed? As far as I can tell it doesn't have an effect on the layout nor design view.

Reply:
Without this it shouldn't align with top bar title.
Becuase I've added marginstart to preference_widget_radiobutton.xml

Reply:
Hmm yep you're right. Removing that line changes nothing because I've added marginstart to preference_widget_radiobutton.xml and CustomizationFragment.kt uses RadioButtonPreference.kt

Reply:
I think `org.mozilla.fenix.settings.RadioButtonPreference` handles setting the layout for you. I tried removing the attribute and the text is still aligned properly.